src/app/events/[slug]/page.tsx

```diff
@@ -35,7 +35,11 @@ export function generateMetadata({ params }: EventProps) {
   const data = getEventData(slug)
 
   return createMetadata({
-    seo: { ...data.seo, image: graphicsData.events1.data.src },
+    seo: {
+      ...data.seo,
+      title: data.seo.title || `${data.title} - Filecoin Event`,
```

Comment:
The reason for the type errors and need for assertion is because we switched to using `SeoMetadataSchemaWithOptionalTitle` but at this we can just keep using `SeoMetadataSchema` - We will already have passed the title.

Reply:
Done.